pupil-spa/src/app/services/check-complete/check-complete.service.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { TestBed } from '@angular/core/testing';
+import { CheckCompleteService } from './check-complete.service';
+import { SubmissionService } from '../submission/submission.service';
+import { SubmissionServiceMock } from '../submission/submission.service.mock';
+import { StorageService } from '../storage/storage.service';
+import { AzureQueueService } from '../azure-queue/azure-queue.service';
+import { AuditService } from '../audit/audit.service';
+import { TokenService } from '../token/token.service';
+import { AppConfigService, loadConfigMockService } from '../config/config.service';
+import { QUEUE_STORAGE_TOKEN } from '../azure-queue/azureStorage';
+import { APP_INITIALIZER } from '@angular/core';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+
+let checkCompleteService: CheckCompleteService;
```

Comment:
Non-blocking: and here - variables in alpha order if possible.

Reply:
@jon-shipley done